### PR TITLE
Various front-end fixes

### PR DESCRIPTION
**Pull request by vz3**
Addressing #622 #623 #626

Implements Favicons, removes references (when it makes sense) to FOIA Hub, changing title of "Federal Agencies" page, tweaks to opengraph.

### CODE REVIEW

File: foia_hub/static/images/favicon/browserconfig.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<browserconfig>
+  <msapplication>
+    <tile>
+      <square70x70logo src="/static/images/favicon/mstile-70x70.png"/>
```
Comment (khandelwal) [inline foia_hub/static/images/favicon/browserconfig.xml]: I don't know enough about what's going on here to understand how this file is used. That said, we'll be using S3 to host our static assets. That means these paths won't work. 

@seanherron or @cmc333333 Have you encountered this before? Is there a standard way to tackle this?

File: foia_hub/templates/includes/head.html [hunk 2]
```diff
@@ -5,6 +5,27 @@
...
+<link rel="icon" type="image/png" href="/static/images/favicon/android-chrome-192x192.png" sizes="192x192">
+<link rel="icon" type="image/png" href="/static/images/favicon/favicon-16x16.png" sizes="16x16">
+<link rel="manifest" href="/static/images/favicon/manifest.json">
+<link rel="shortcut icon" href="/static/images/favicon/favicon.ico">
+<meta name="msapplication-TileColor" content="#ffc40d">
+<meta name="msapplication-TileImage" content="/static/images/favicon/mstile-144x144.png">
+<meta name="msapplication-config" content="/static/images/favicon/browserconfig.xml">
+<meta name="theme-color" content="#ffffff">
+
 <script src="{{ static("js/jquery-2.1.1.min.js") }}"></script>
```
Comment (khandelwal) [inline foia_hub/templates/includes/head.html]: See how this link refers to a static asset? You'll want to do the same thing for the image paths above. This'll ensure that Django will use the correct path to the static images.

Comment (vz3) [inline foia_hub/templates/includes/head.html]: so it'll be:
`{{ static("images/favicon/apple-touch-icon-120x120.png") }}`
correct?

**Comment by vz3** [comment]
@khandelwal I (believe) I've addressed the issues in e0d4626 and f72e977

File: foia_hub/templates/includes/head.html (hunk 2 above)
Comment (khandelwal) [inline foia_hub/templates/includes/head.html]: Yes, I believe so.

**Comment by khandelwal** [comment]
Here's an example: https://github.com/18F/letgirlslearn/blob/master/rendered/index.html#L38-L58
for where it would be in the template. 

(If we do it in the template, we can use those static() links and things will work).

**Comment by vz3** [comment]
The lines from that example are the same as what's already in `head.html`

**Pull request merged**